python/GafferVDBUI/VolumeScatterUI.py

```diff
@@ -0,0 +1,146 @@
+##########################################################################
+#
+#  Copyright (c) 2020, Don Boogert. All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are
+#  met:
+#
+#      * Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials provided with
+#        the distribution.
+#
+#      * Neither the name of Don Boogert nor the names of
+#        any other contributors to this software may be used to endorse or
+#        promote products derived from this software without specific prior
+#        written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+#  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+#  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+#  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+#  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+#  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+
+import GafferUI
+import GafferVDB
+
+GafferUI.Metadata.registerNode(
+
+	GafferVDB.VolumeScatter,
+
+	'description',
+	"""
+	Scatter points into active voxels of VDB grid
+	""",
+
+	"layout:activator:density", lambda node : node["mode"].getValue() == GafferVDB.VolumeScatter.Mode.Density,
+	"layout:activator:pointCount", lambda node : node["mode"].getValue() == GafferVDB.VolumeScatter.Mode.PointCount,
+
+	plugs = {
+
+		"name" : [
+
+			"description",
+			"""
+			The name given to the object generated -
+			this will be placed under the parent in
+			the scene hierarchy.
```

Comment:
I know this came from the Scatter node, but I think we could do better :

- Refer to a PointsPrimitive rather than "the object generated"
- Talk about `destination` rather than "the parent"